odata-parser.ometajs

```diff
@@ -32,6 +32,39 @@ var methods = {
 	year: 1
 };
 
+var operatorPrecedence = {
+	or: 0,
+	and: 0,
+	eq: 1,
+	ne: 1,
+	gt: 1,
+	ge: 1,
+	lt: 1,
+	le: 1,
+	sub: 2,
+	add: 3,
+	mod: 4,
+	div: 5,
+	mul: 6
+}
+
+// v4 operator precedence
+// var operatorPrecedence = {
```

Comment:
Are these backwards incompatible? They look saner to me and closer to operator precedence in other languages.

Reply:
They would definitely change the behaviour in some cases because the precedence would change - whether it would actually affect any uses I don't know, but it definitely had the potential to